texparser/tar_extractor.py

```diff
@@ -13,28 +15,34 @@ def extract_folder(self, folder, func):
         filenames = os.listdir(folder)
         for filename in filenames:
             try:
-                processes.append(func(folder + "/" + filename))
-            except:
+                path = Path(folder) / filename
+                filename, file_extension = os.path.splitext(path)
+                processes.append(func(filename + file_extension))
+            except FileExistsError:
+                sys.stderr.write("Error message: File already exists. \n")
                 pass  # How should we handle already existing folders?
         return processes
-
     def extract_file_into_folder(self, file_name: str, params: str, ending: str) -> tuple:
         tar_process = any
         if file_name.endswith(ending):
             if ending in file_name:
-                file_folder_path = self.__create_file_folder_path(
```

Comment:
Why is it no longer private is that ment to be like that? 
For testing you could make the method protected

Reply:
changed it to be protected (e.g. have one underscore instead of two or none)

protected functions can be accessed for tests, but private ones (with two underscores) cannot